tests/unit/_internal/bento/simplebento/.bentomlignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
not related to this PR - one thing I noticed when building bento with 1.0 was lots of .pyc files and __pycache__ directory was included in the final Bento. Do you think we should add those to the default exclude list?